go/filemanager/pfsmod/chunk.go

```diff
@@ -0,0 +1,118 @@
+package pfsmod
+
+import (
+	"errors"
+	"fmt"
+	log "github.com/golang/glog"
+	"io"
+	"net/url"
+	"os"
+	"strconv"
+)
+
+//ChunkCmd structure
+type ChunkCmd struct {
```

Comment:
I think we need a comment that describe what is ChunkCmd, I am confused why there is "cmd", from the definition, I can not see what "cmd" does?